src/js/subtraction/components/__tests__/Subtraction.test.tsx

```diff
@@ -86,9 +87,10 @@ describe("<Subtraction />", () => {
     });
 
     it("should render /subtractions/:subtractionId route", async () => {
-        history.push(`/subtractions/${subtractionMinimal.id}`);
+        const subtraction = createFakeSubtraction();
+        history.push(`/subtractions/${subtraction.id}`);
```

Comment:
Not a change you made, but looks like the same history object is being used in every test in this module which is bad from a test isolation perspective. A new history object should be made before every test via `beforeEach`